Index: src/main/java/com/drivecheaper/Kunden.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/drivecheaper/Kunden.java b/src/main/java/com/drivecheaper/Kunden.java
deleted file mode 100644
--- a/src/main/java/com/drivecheaper/Kunden.java	(revision 6edc21834d7dce28b8837d530cf72700a4a5b639)
+++ /dev/null	(revision 6edc21834d7dce28b8837d530cf72700a4a5b639)
@@ -1,5 +0,0 @@
-package com.drivecheaper;
-
-public abstract class Kunden {
-
-}
Index: .gitignore
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1773759548046)
+++ b/.gitignore	(date 1773759548046)
@@ -0,0 +1,39 @@
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
+.kotlin
+
+### IntelliJ IDEA ###
+.idea/modules.xml
+.idea/jarRepositories.xml
+.idea/compiler.xml
+.idea/libraries/
+*.iws
+*.iml
+*.ipr
+
+### Eclipse ###
+.apt_generated
+.classpath
+.factorypath
+.project
+.settings
+.springBeans
+.sts4-cache
+
+### NetBeans ###
+/nbproject/private/
+/nbbuild/
+/dist/
+/nbdist/
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
+
+### Mac OS ###
+.DS_Store
\ No newline at end of file
Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1773759564272)
+++ b/.idea/.gitignore	(date 1773759564272)
@@ -0,0 +1,10 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Ignored default folder with query files
+/queries/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module com.drivecheaper.drivecheaper {\r\n    requires javafx.controls;\r\n    requires javafx.fxml;\r\n\r\n    requires org.controlsfx.controls;\r\n    requires com.dlsc.formsfx;\r\n    requires net.synedra.validatorfx;\r\n    requires org.kordamp.ikonli.javafx;\r\n    requires org.kordamp.bootstrapfx.core;\r\n    requires java.sql;\r\n\r\n    opens com.drivecheaper to javafx.fxml;\r\n    exports com.drivecheaper;\r\n    exports com.drivecheaper.Fahrzeuge;\r\n    opens com.drivecheaper.Fahrzeuge to javafx.fxml;\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	(revision 6edc21834d7dce28b8837d530cf72700a4a5b639)
+++ b/src/main/java/module-info.java	(date 1774005135672)
@@ -11,6 +11,8 @@
 
     opens com.drivecheaper to javafx.fxml;
     exports com.drivecheaper;
-    exports com.drivecheaper.Fahrzeuge;
-    opens com.drivecheaper.Fahrzeuge to javafx.fxml;
+    exports com.drivecheaper.fahrzeuge;
+    opens com.drivecheaper.fahrzeuge to javafx.fxml;
+    exports com.drivecheaper.kunde;
+    opens com.drivecheaper.kunde to javafx.fxml;
 }
\ No newline at end of file
